docs/unified/novacustom/firmware-update.md

```diff
@@ -4,6 +4,14 @@ The following documentation describes the process of Dasharo open-source
 firmware update. The update process may be different, depending on which
 firmware version is currently installed on your device.
 
+This process can be performed using the Dasharo Tools Suite available in the
+Dasharo Pro Package (formerly Dasharo Entry Subscription) instead. For
+instructions on how to use the DTS instead check
+
+This process can be performed using the Dasharo Tools Suite included in the Dasharo
+Pro Package subscription (formerly Dasharo Entry Subscription) instead. For
+instructions on how to use the DTS see the [DTS documentation](../../dasharo-tools-suite/documentation.md#firmware-update)
```

Comment:
Partially duplicated

Reply:
Oops, this shouldn't have been changed. My guess is it's because of the mandatory and impossible to disable middle mouse button shortcut on gnome which pastes from the clipboard. It's really easy to use it accidentally on a touch pad.

https://github.com/Dasharo/docs/compare/45e632d7c45be0ff999901abb679c5bfbcacd976..782894f56ca30ef10d0eb358a84a6cf0f1f3909b

Reply:
@philipandag It can be disabled in `Tweaks` software:
![image](https://github.com/user-attachments/assets/5d310de8-9a0f-49c6-931a-3274dfd82e34)

Reply:
Though from my testing it seems that it also depends on configuration on other applications. In Firefox it's `middlemouse.paste` in `about:config`, in `~/.tmux.conf` I also have custom configuration:

```
# paste from system clipboard
bind-key -n MouseDown2Pane run "tmux set-buffer \"$(xclip -o -sel clipboard)\"; tmux paste-buffer"
```

So you need to check your editors configuration if you want to disable it completely.

Reply:
That's right, I have misspoke. Many applications implement this shortcut themself and don't care about gnome tweaks. That's why it's "impossible" to easily disable it globally. Thanks for the tips :)